pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -4552,6 +4555,75 @@ private CompletableFuture<Void> createSubscriptions(TopicName topicName, int num
         return result;
     }
 
+    /**
+     * It creates subscriptions for new partitions of existing partitioned-topics.
+     *
+     * @param topicName     : topic-name: persistent://prop/cluster/ns/topic
+     * @param numPartitions : number partitions for the topics
+     *
+     */
+    private CompletableFuture<Void> createMissedSubscriptionsAsync(TopicName topicName, int numPartitions) {
+        CompletableFuture<Void> result = new CompletableFuture<>();
+        PulsarAdmin admin;
+        try {
+            admin = pulsar().getAdminClient();
```

Comment:
This topic should be on this broker, we don't need call admin api to get what you need.

Reply:
`org.apache.pulsar.broker.admin.impl.PersistentTopicsBase#internalCreateMissedPartitions` does not call `validateTopicOwnershipAsync` to verify whether the topic belongs to the broker, but only to determine whether the topic belongs to the namespace of the broker, so I have reserved the logic of using admin to create subscriptions in `org.apache.pulsar.broker.admin.impl.PersistentTopicsBase#createMissedSubscriptionsAsync`~